torch/_inductor/fx_passes/quantization.py

```diff
@@ -656,6 +658,15 @@ def __init__(self, op_name: str, scalars_attr=None, algorithm_attr=None):
                 ),
                 dtype=original_pattern_output_dtype,
             ),
+            UnaryAttr("swish", [], ""): generate_pattern_with_output_quant(
+                _unary_fusion_pattern(
+                    _silu_fusion,
+                    get_dequantize_qconv_pt2e_pattern(1 if is_bf16 else 2),
+                    2,
+                    is_bf16,
```

Comment:
How about other fusions above?

Reply:
Updated for `hardsiwsh` and `hardtanh` https://github.com/pytorch/pytorch/pull/122374